### PR TITLE
METRON-755 Update GitHub PR Template

### PR DESCRIPTION
**Pull request by JonZeolla**
Thank you for submitting a contribution to Apache Metron (Incubating).  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification guildlines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel). 
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

## Contributor Comments
This is a PR to make some minor changes to the PR template.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
-Thank you for submitting a contribution to Apache Metron (Incubating).
-Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.
-Please refer also to our [Build Verification guildlines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.
```
Comment (cestella) [inline .github/PULL_REQUEST_TEMPLATE.md]: I dunno, I kinda prefer guildlines ;)

**Comment by james-sirota** [comment]
+1 i agree thats a great idea. Should this be on the wiki or version controlled?  The other two docs this refers to are on the wiki. we should probably be consistent. either transition them into github or have this be a part of the wiki

**Comment by cestella** [comment]
The benefit of having this in git is that when you go to https://github.com/apache/incubator-metron/pulls, committers can see in one screen which PRs have passing travis and how far along in the checklist people are.

**Comment by JonZeolla** [comment]
I generally like the idea of us moving wiki pages to git except for specific cases.  I put up a discuss thread on the topic a little while back.  https://lists.apache.org/thread.html/17904a10b8f41a97dfac84dcb3c22a1a0229fa9db170fa762741c2fe@%3Cdev.metron.apache.org%3E

**Comment by JonZeolla** [comment]
It seems that some people prefer to add their comments to the top instead of the bottom.  I have no preference.  Would anybody recommend we swap it?

**Comment by justinleet** [comment]
I prefer top, but I don't really care that much.

**Comment by kylerichardson** [comment]
I'd prefer contributor comments at the top.

**Comment by JonZeolla** [comment]
Ok, moved it to the top.

**Pull request closed without merging**